src/app/src/components/QuizNavbar.js

```diff
@@ -30,12 +100,52 @@ const CloseButtonPlaceholder = styled(Box)`
 const QuizNavbar = props => {
     const { dispatch, question, results, isQuestionActive } = props;
 
+    const check = (
+        <CorrectIcon>
+            <FontAwesomeIcon
+                icon={['fal', 'check']}
+                size='3x'
+                className='icon'
+            />
+        </CorrectIcon>
+    );
+    const times = (
+        <IncorrectIcon>
+            <FontAwesomeIcon
+                icon={['fal', 'times']}
+                size='3x'
+                className='icon'
+            />
+        </IncorrectIcon>
+    );
+
     const icons = [0, 1, 2, 3, 4].map(idx => {
         const result = results[idx];
+        const isCurrent = question === idx;
         const correct = result && result.numWrong < 2;
+        const incorrect = result && !correct;
+        function setIconTheme() {
```

Comment:
Nice work with this! Scope 🌈

Consider rewriting as an arrow function for consistency
```
const setIconTheme = () => {};
```